substrate/frame/collective/src/lib.rs

```diff
@@ -768,7 +802,10 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 			}
 		} else {
 			if position_no.is_none() {
-				voting.nays.push(who.clone());
+				voting
+					.nays
+					.try_push(who.clone())
+					.expect("Proposal voting nays can't overflow; qed");
```

Comment:
Why can't they overflow?

Reply:
same here:

> the bound was previously enforced via code, since there could be at most `members_count` votes, and `members_count` is at most `MaxMembers`